binplace.targets

```diff
@@ -35,6 +35,9 @@
       <TargetingPackDir Include="$(RefPath)" />
       <TargetingPackDir Condition="'$(IsNETCoreAppRef)' == 'true'" Include="$(BinDir)netcoreapp\pkg\ref" />
     </ItemGroup>
+    <ItemGroup Condition="$(NuGetTargetMoniker.Contains('UAP')) Or $(NuGetTargetMoniker.Contains('NETStandard'))">
```

Comment:
I agree it is very hard coded now, but I'm not sure of how to implement it in a better way now. Do you want that fix to happen before I merge this PR? or is it ok to log an issue for that as an enhancement.